app/(app)/alpha/settings/page.tsx

```diff
@@ -30,14 +33,21 @@ export default async function Page() {
     redirect("/get-started");
   }
 
-  const user = await prisma.user.findUnique({
-    where: {
-      id: session.user.id,
+  const existingUser = await db.query.user.findFirst({
```

Comment:
@John-Paul-Larkin  I believe this is your work so would love your input on this. If theres anything we can improve